packages/grid/src/vaadin-grid-column.d.ts

```diff
@@ -3,92 +3,15 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { GridDefaultItem, GridItemModel } from './vaadin-grid.js';
 
-export type GridBodyRenderer<TItem> = (
-  root: HTMLElement,
-  column: GridColumn<TItem>,
-  model: GridItemModel<TItem>,
-) => void;
+import type { GridDefaultItem } from './vaadin-grid.js';
+import type { GridColumnMixin } from './vaadin-grid-column-mixin.js';
+import type { GridBodyRenderer as GridMixinBodyRenderer } from './vaadin-grid-column-mixin.js';
+import type { GridHeaderFooterRenderer as GridMixinHeaderFooterRenderer } from './vaadin-grid-column-mixin.js';
```

Comment:
All of these are imported from the same file, so could be changed to this:

```suggestion
import type {
  GridBodyRenderer as GridMixinBodyRenderer,
  GridColumnMixin,
  GridHeaderFooterRenderer as GridMixinHeaderFooterRenderer,
} from './vaadin-grid-column-mixin.js'
```

Reply:
Thx, done